modules/support-reviewing-an-access-request-from-the-hybrid-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-the-hybrid-cloud-console_{context}"]
+= Reviewing an access request from the {hybrid-console-second}
+
+.Prerequisites
```

Comment:
This should be inside the below ifdef as well. It doesn't seem to render the heading if there's nothing below it, but it shouldn't be stranded here just in case.

Also, I don't see this file being included anywhere that isn't ROSA - is there a reason for the below conditional, if it's not actually being used currently?